### ISSUE TITLE
Pandoc LaTeX/PDF Writer Not Removing Header Numbering at Header Level 4

### ISSUE THREAD

**Issue by michael-okeefe** [jgm/pandoc#3058]
**Objective**: Remove numbering for headers in a markdown document

**What I Did**: Annotated ATX style markdown headers with "{-}" (also tried "{.unnumbered}"). The pandoc arguments at the command line are:

``` bash
pandoc --parse-raw --smart --normalize --standalone --wrap=none --table-of-contents --toc-depth=3 --latex-engine=xelatex -Vgeometry="margin=1in" --number-sections -o junk.pdf junk.md
```

**Expected Result**: Headers would appear in output LaTeX/PDF without outline numbering.

**Actual Result**: Headers less than level 4 are as expected. Headers of level 4 and greater appear in output along with their outline numbers and a "*" between the outline number and the header text.

_Minimum Example_:

File `junk.md`: 

``` markdown
# A {-}

a

## AA {-}

aa

### AAA {-}

aaa

#### AAAA {-}

aaaa

##### AAAAA {-}

aaaaa
```

Output LaTeX (snippet):

``` latex
\begin{document}

{
\setcounter{tocdepth}{3}
\tableofcontents
}
\section*{A}\label{a}
\addcontentsline{toc}{section}{A}

a

\subsection*{AA}\label{aa}
\addcontentsline{toc}{subsection}{AA}

aa

\subsubsection*{AAA}\label{aaa}
\addcontentsline{toc}{subsubsection}{AAA}

aaa

\paragraph*{AAAA}\label{aaaa}
\addcontentsline{toc}{paragraph}{AAAA}

aaaa

\subparagraph*{AAAAA}\label{aaaaa}
\addcontentsline{toc}{subparagraph}{AAAAA}

aaaaa

\end{document}
```

_Screenshot of PDF_

![Screenshot of Output PDF](https://cloud.githubusercontent.com/assets/1099583/17414685/9ee42702-5a43-11e6-9d6b-278ee7b911b7.png)

**Comment by jkr** [comment]
@jgm: There is one ugly way to take care of this: setting the secnumdepth to 0 for each of the paragraph and subparagraph, and then resetting it after:

~~~latex
...

\subsubsection*{AAA}\label{aaa}
\addcontentsline{toc}{subsubsection}{AAA}

aaa

\edef\foo{\arabic{secnumdepth}}
\setcounter{secnumdepth}{0}
\paragraph{AAAA}\label{aaaa}
\addcontentsline{toc}{paragraph}{AAAA}
\setcounter{secnumdepth}{\foo}

aaaa

\edef\foo{\arabic{secnumdepth}}
\setcounter{secnumdepth}{0}
\subparagraph{AAAAA}\label{aaaaa}
\addcontentsline{toc}{subparagraph}{AAAAA}
\setcounter{secnumdepth}{\foo}

aaaaa

~~~

The outcome is as it should be, and it doesn't affect other headers. 

The nice things about this approach is that it doesn't require anything in the preamble, and it's self-contained (i.e., no header bothers any other). It will also be very easy to put into the writer. The downside is that it's ugly.

What do you think?

**Comment by jkr** [comment]
I should also add that its ugliness is slightly mitigated by the rareness of its use (4th- and 5th-level headers with an unnumbered class with numbering specified and no changes made to the template).

**Comment by tarleb** [comment]
Appears to be fixed as of pandoc 2.13. Hence closing.
